packages/slate-sections-plugin/README.md

```diff
@@ -74,91 +74,3 @@ In order to have separate schema files, you must create a folder for the section
 ```
 
 This will create an `article-template.liquid` with the contents of the `template.liquid` file and with JSON wrapped in `{% schema %}` tags, which will be copied over to the `./dist` sections folder.
-
-## Translations
```

Comment:
If we're getting rid of this section we probably still want to specify how you can have a locales folder since right now it just shows having a schema.json and template.liquid in a folder, and a sections folder with just liquid files.

Reply:
Good call - I'll bring back this section but instead write a small description and point to the `@shopify/slate-translations` README.